approx_dates/models.py

```diff
@@ -6,6 +6,23 @@
 
 import six
 
+class classproperty(object):
+    """
+    This decorator acts as @property (returning a new result
+    from the method each time the property is requested) but
+    for what would otherwise be a @classmethod.  
+    
+    source: http://stackoverflow.com/questions/128573/us
```

Comment:
It's not ideal to have a linebreak in the URL - you can get a more compact version via the share link on StackOverflow: http://stackoverflow.com/a/13624858/223092